compiler/luci/export/src/CircleBuiltinTypesExtractor.h

```diff
@@ -118,6 +118,10 @@ class BuiltinOptionsExtractor final
     return circle::CreateCosOptions(_builder).Union();
   }
   flatbuffers::Offset<void> visit(luci::CircleCustom *) { return _no_option; }
+  flatbuffers::Offset<void> visit(luci::CircleCumSum *node)
```

Comment:
@batcheu , after landing this code, found that alphabetical order is not correct. please post a fix for this.

Reply:
Okay, I'll fix it. Thanks.